File: WantList.Core/Anime.cs
using System;

namespace WantList.Core
{
    public class Anime
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int? AnidbId { get; set; }
        public DateTime AddedDateTime { get; set; }
        public DateTime ReleaseDate { get; set; }
        public AnimeType Type { get; set; }
        public int EpisodeCount { get; set; }
        public int WantRank { get; set; }
        public bool Redownload { get; set; }
        public bool BluRay { get; set; }
        public Quality Quality { get; set; }
        public string BluRayRelease { get; set; }
        public bool Deleted { get; set; }
        public byte[] Image { get; set; }
    }
}

File: WantList.Angular/src/app/entities/anime.ts
import { AnimeType } from './anime-type.enum';
import { Quality } from './quality.enum';

export class Anime {
  id: number;
  name: string;
  anidbId: number;
  addedDateTime: Date;
  releaseDate: Date;
  type: AnimeType;
  episodeCount: number;
  wantRank: number;
  redownload: boolean;
  bluRay: boolean;
  quality: Quality;
  bluRayRelease: string;
  deleted: boolean;

  public static copyFrom( from: Anime, to: Anime ): void {
    Object.keys(from).forEach(key => to[key] = from[key]);
  }
}
